Source/SwiftLintFramework/Rules/JoinedDefaultRule.swift

```diff
@@ -49,10 +49,14 @@ public struct JoinedDefaultParameterRule: ASTRule, ConfigurationProviderRule, Op
 
     private func defaultSeparatorOffset(dictionary: [String: SourceKitRepresentable], file: File) -> Int? {
         guard
-            let bodyOffset = dictionary.bodyOffset,
-            let bodyLength = dictionary.bodyLength else { return nil }
+            let argument = dictionary.substructure.first(where: { $0.name == "separator" }),
+            let argumentBodyOffset = argument.bodyOffset,
```

Comment:
I've just checked in another version which check the number of arguments and the argument name.